ioctl/cmd/contract/contractdeploysol.go

```diff
@@ -76,6 +76,10 @@ func contractDeploySol(args []string) error {
 
 	for name := range contracts {
 		if strings.HasSuffix(name, contractName) {
+			if contractName != args[0] {
+				return output.NewError(output.CompilerError,
+					fmt.Sprintf("there are more than one %s contract", args[0]), nil)
+			}
```

Comment:
the items of "contracts" are in "file:contract" format? this is a hidden knowledge, which should be added in some comment.

Reply:
So I changed the usage of this command. "file:contract" format is optional.

Reply:
This feature is done by solc whose output is like:
```
➜  ioctl contract compile Multisend
Output:
======= multisend2.sol:Multisend =======
Binary:
0x608060405234801561001057600080fd5b5060405160408061096483398101604052805160209091015160008054600160a060020a03191633179055600191909155600255610911806100536000396000f30060806...
Contract JSON ABI
[{"constant":false,"inputs":[{"name":"recipients","type":"address[]"},{"name":"amounts","type":"uint256[]"},{"name":"payload","type":"string"}],"name":"sendCoin","outputs":[],"payable":true,"stateMutability":"payable","type":"function"},
...
```

`multisend2.sol:Multisend` is the contract name output by solc. I didn't realized to use this feature when writing this command.